src/components/Header/Header.scss

```diff
@@ -0,0 +1,5 @@
+.header {
+  background-color: var(--roxoClaro);
+  background-color: 7e798c;
```

Comment:
nao manjo do sass, mas essa linha é assim mesmo? e tem outra linha de background-color em cima.

So uma duvida mesmo, realmente nao manjo do sass

Reply:
Se for isso ai mesmo, o resto dah de boa pra me 😺

Reply:
É, não tá faltando um # na cor de fallback?

Reply:
Obrigado <3